tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/UpgradePlanCorePlugin.java

```diff
@@ -43,10 +74,54 @@ public static IStatus createErrorStatus(String msg, Exception e) {
 		return new Status(IStatus.ERROR, ID, msg, e);
 	}
 
+	public static IUpgradePlanOutline getFilterOutlines(String name) {
```

Comment:
@simonjhy @DL-EthanSun  we don't need to support multiple ways of getting outlines.  We should only support "offline".  we don't need to consider anything else.  Can you please remove this method?